### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by proteus1121**

### CODE REVIEW

File: charts/build.gradle [hunk 1]
```diff
@@ -0,0 +1,15 @@
+dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter-actuator'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    implementation "org.springframework.boot:spring-boot-starter-tomcat"
```
Comment (leafchild0) [inline charts/build.gradle:5]: How about we use Netty. It's more lightweight and async friendly

File: charts/src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,7 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/charts
+spring.datasource.username=root
+spring.datasource.password=test
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+spring.jpa.database-platform = org.hibernate.dialect.MySQL5Dialect
```
Comment (leafchild0) [inline charts/src/main/resources/application.properties:5]: Postgres was in the list, no?

**Review by leafchild0 — DISMISSED** [review]
This looks good to me, suggested few changes

File: charts/build.gradle (hunk 1 above)
Comment (proteus1121) [reply charts/build.gradle]: ok, lets use web flux, it has netty out of box, hope we will use it.

File: charts/src/main/resources/application.properties (hunk 2 above)
Comment (proteus1121) [reply charts/src/main/resources/application.properties]: yes, changed

File: charts/build.gradle (hunk 1 above)
Comment (leafchild0) [reply charts/build.gradle]: Sure, great idea, it's async and a bit better than endless amounts of CompletableFuture

**Review by leafchild0 — APPROVED** [review]

**Pull request merged**